src/types.ts

```diff
@@ -13,7 +13,16 @@ export type ThirdPartyTag = {
 	loaded?: boolean;
 	onLoad?: () => void;
 	shouldRun: boolean;
+	sourcepointId?: undefined;
 	name?: string;
 	url?: string;
 	useImage?: boolean;
 };
+
+declare global {
+	interface Window {
+		_brandmetrics?: { cmd: string; val: Record<string, unknown> }[];
+		// eslint-disable-next-line no-undef
```

Comment:
what is this erroring on? we should be wary of disabling rules

Reply:
ESLint is unhappy because `googletag` is not defined… If we make it happy by import it from the declaration file type as below, TS is then unhappy because you cannot import from a declaration file.

```ts
import googletag from '@types/doubleclick-gpt';
```

Reply:
Following a suggestion from @SiAdcock, declaring it as a `readonly` global in `.eslintrc` did the trick:

```json
{
	"globals": { "googletag": "readonly" },
	# … rest of config
}
```